src/corelibs/U2View/src/ov_msa/MSAEditor.cpp

```diff
@@ -746,4 +758,34 @@ void MSAEditor::sl_convertBetweenDnaAndRnaAlphabets() {
     msaObject->replaceAllCharacters(fromChar, toChar, resultAlphabet);
 }
 
+void MSAEditor::sl_convertRawToDnaAlphabet() {
+    CHECK(!maObject->isStateLocked(), )
+
+    auto alphabetId = maObject->getAlphabet()->getId();
+    CHECK(alphabetId == BaseDNAAlphabetIds::RAW(), );
+
+    auto msaObject = getMaObject();
+    auto alphabetRegistry = AppContext::getDNAAlphabetRegistry();
+    U2OpStatus2Log os;
+    U2UseCommonUserModStep userModStep(msaObject->getEntityRef(), os);
```

Comment:
`Q_UNUSED(userModStep);`

Reply:
Q_UNUSED is required here to avoid warning.